src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
+  <mat-card>
+    <form [formGroup]="newOfficeForm" (ngSubmit)="submit()">
+      <mat-card-content>
+        <div fxLayout="column" fxLayoutAlign="center">
+          <mat-form-field fxFlex="100%">
+            <mat-label>Office</mat-label>
+            <input required matInput formControlName="name" />
```

Comment:
```suggestion
            <input required matInput formControlName="name">
```
Remove the `/` from all inputs.